lightly_studio/src/lightly_studio/models/dataset.py

```diff
@@ -32,6 +32,7 @@ class DatasetView(DatasetBase):
     dataset_id: UUID
     created_at: datetime
     updated_at: datetime
+    total_sample_count: int | None = None
```

Comment:
This is breaking a pattern that we have. See "SampleViewsWithCount" and "AnnotationViewsWithCount" - here to follow the pattern we'd need "DatasetViewWithCount".

Reply:
Addressed